release-notes.txt

```diff
@@ -1,3 +1,543 @@
+#12371: fail to send data to can bus
+#12057: ESP32 + DHT + SAUL reading two endpoints causes freeze.
+#12045: floats and doubles being used all over the place.
+#11104: STM32: SPI clock not returning to idle state and generating additional
+        clock cycles
+#9419: cpu/msp430: GPIO driver doesn't work properly
+#4876: at86rf2xx: Simultaneous use of different transceiver types is not
+       supported
+
+Native related issues (3)
+-------------------------
+#13501: SDL2 does not work due to missing getpid
+#5796: native: tlsf: early malloc will lead to a crash
+#495: native not float safe
+
+Other platforms related issues (20)
+-----------------------------------
+#14822: MPU doesn't work on cortex-m0+
+#14728: stm32f7: Large performance difference between stm32f746 and stm32f767
+#14679: examples/gnrc_border_router: esp_wifi crashes on disconnect
+#14572: tests/mpu_noexec_ram: fails on  i-nucleo-lrwan1
+#14410: MIPS: toolchain objcopy doesn't work and no .bin can be generated
+#14237: esp32-wroom-32: tests/netstats_l2 failing sometimes
+#14015: stm32152re: hardfault when DBGMCU_CR_DBG* bits are set and branch after
+        __WFI()
+#13606: gcoap/esp8266: Stack overflow with gcoap example
+#13390: Cannot use LLVM with Cortex-M boards
+#13104: boards/hifive1: flashing issue
+#13086:  Failing tests on FE310 (Hifive1b)
+#12763: [TRACKING] Fixes for automatic tests of ESP32 boards.
+#12651: Failing tests on AVR (tested with atmega256rfr2-xpro)
+#12168: pkg/libb2: blake2s doesn't work on AVR
+#11885: arm7: printf() with float/double not working
+#10258: Incorrect defaultPORT building for esp32-wroom-32 on macOS
+#10122: Indeterministic hard fault in _mutex_lock(), with nRF52 SoftDevice
+#10076: cpu/cortexm_common: irq_enable returns the current state of interrupts
+        (not previous)
+#6567: periph/spi: Switching between CPOL=0,1 problems on Kinetis with software
+       CS
+#4954: chronos: compiling with -O0 breaks
+
+Build system related issues (14)
+--------------------------------
+#15185: build: info-build doesn't work with boards without port set
+#14504: BUILD_IN_DOCKER ignores USEMODULE
+#14288: pkg/ccn-lite: unable to `make -j -C examples/ccn-lite-relay`
+#14264: make: ccache leads to differing binaries
+#13492: make -j flash broken on esp* (will always flash the previous binary)
+#12771: dist/tools/cppcheck/cppchck.sh: errors when running with Cppcheck 1.89
+#10850: Tracking: remove harmful use of `export` in make and immediate
+        evaluation
+#9913: Build dependencies - processing order issues
+#9742: `buildtest` uses wrong build directory
+#9645: Different build behavior between `murdock` and `riot/riotbuild:latest`
+       image
+#8913: make: use of immediate value of variables before they have their final
+       value
+#8122: doxygen: riot.css modified by 'make doc'
+#6120: Windows AVR Mega development makefile Error
+#4053: macros: RIOT_FILE_RELATIVE printing wrong file name for headers
+
+Other issues (41)
```

Comment:
Some issues in this category could be moved to some of the categories above. They are just missing the right labels.
See [this line](https://github.com/RIOT-OS/RIOT/pull/15340/files#diff-f41b6887f65b706736aec3e6716bce4844a2aa169cba5e675c630f10b4fdbcecR416) which could be moved to `Other platforms related issues`

Reply:
Blame the script :)

Reply:
No I blame the maintainers who don't set the labels ;)

Reply:
> No I blame the maintainers who don't set the labels ;)

Please don't, most, if not all, of the issues are labelled. [The script is just a tad strict in which issues](https://github.com/RIOT-OS/RIOT-release-manager/blob/master/riot_release_manager.py#L557-L595) are sorted where.

Reply:
Maybe we should blame the author of the script :stuck_out_tongue_winking_eye: